game/templates/game/leaderboard.html

```
@@ -1,3 +1,4 @@
+{% load static %}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)